change_log/next/FE-2450-Add_src_prop_to_Profile_component.yml

```
@@ -0,0 +1 @@
+New Features: "adds support for src prop in Profile image. (component: Profile)"
```

The RFC for semantic release dictates that we use the the imperative, present tense, so please could you do this in the commit message and it makes sense to be consistent here.
```suggestion
New Features: "add support for src prop in Profile image. (component: Profile)"
```